src/main/java/org/tron/common/runtime/vm/program/ContractState.java

```diff
@@ -42,7 +42,7 @@
   private final DataWord address;  // contract address
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this trailing comment on the previous empty line. ([squid:TrailingCommentCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3ATrailingCommentCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1664&issues=AWbKHCyUO08i52vdhEhX&open=AWbKHCyUO08i52vdhEhX)

[ISSUE_KEY]: AWbKHCyUO08i52vdhEhX